packages/react/examples/react-router/package.json

```diff
@@ -7,10 +7,12 @@
     "carbon-components-react": "latest",
     "carbon-icons": "latest",
     "node-sass": "^4.11.0",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.3"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "history": "^5.0.0",
```

Comment:
Why is `history` added?

Reply:
Without history, the `react-router-dom` didn't work and gave dependancy error. So added that.